extensions/job_management/standard/sections/clause_6_job_management.adoc

```diff
@@ -0,0 +1,257 @@
+|`412` |The status code indicates that one or more conditions given in the request header fields evaluated to false when tested by the server.
+|`413` |The server is refusing to process the request because the request content is larger than the server is willing or able to process.
+|`415` |The server is refusing to service the request because the content is in a format not supported by this method on the target resource.
+|`422` |The server understands the content type of the request content and the syntax of the request content is correct, but was unable to process the contained instructions. For example, the submitted resource does not meet a semantic constraint, e.g. a mandatory property is missing.
+|`500` |An internal error occurred in the server.
+|===
+
+NOTE: Status code `422` is not yet an official HTTP status code, but is expected to be added by the https://www.ietf.org/archive/id/draft-ietf-httpbis-semantics-14.html#name-422-unprocessable-content[draft IETF RFC "HTTP Semantics"].
+
+More specific guidance is provided for each resource, where applicable.
+
+include::../recommendations/job-management/PER_additional-status-codes.adoc[]
+
+The API Description Document describes the HTTP status codes generated by that API imeplementation instance. This should not be an exhaustive list of all possible status codes. It is not reasonable to expect an API designer to control the use of HTTP status codes which are not generated by their software. Therefore, it is recommended that the API Description Document be limited to describing HTTP status codes relevant to the proper operation of the API application logic. Client implementations should be prepared to receive HTTP status codes in addition to those described in the API Description Document.
+
+[[cross_origin]]
+==== Cross-origin requests
+
+See <<OAFeat-1,OGC API - Features - Part 1: Core>>, section link:http://www.opengis.net/doc/IS/ogcapi-features-1/1.0#cross_origin[Support for cross-origin requests], about the importance of supporting cross-origin requests, typically via link:https://en.wikipedia.org/wiki/Cross-origin_resource_sharing[Cross-origin resource sharing (CORS)].
+
+
+
+[[job-management-create]]
+=== Creating a new job
+
+==== Sequence diagram
+
+The following diagram illustrates the sequence diagram for deploying
+a new process to the API:
+
+
+```
+Client                                                        Server
+  |                                                             |
+  |  POST /jobs   HTTP/1.1                                      |
+  |  Content-Type: application/json                             |
+  |                                                             |
+  |------------------------------------------------------------>|
+  |                                                             |
+  |  HTTP/1.1 201 Created                                       |
+  |  Location: /jobs/{jobId}                                    |
```

Comment:
In openEO we also return a hreader that returns the OpenEO-Identifier header, which just includes the ID.
We can't reuse this header here due to the "OpenEO-" prefix, but maybe a generic X-OGC-Identifier or so would work? Is this a valuable information to have?

Reply:
What could be considered is a combination of `Content-Location` and `Content-ID`. 

I don't see an issue with adding those, but I don't think they would be required, since the information can be obtained from the response contents (it is assumed here that job `status`+`id` are returned, as per other comments).

Reply:
Content-Location is weird as there's already the standardized Location header.

Reply:
`Location` is to indicate where to redirect to. `Content-Location` is to describe where the contents of the resource can be found. In this case, both URI are the same, but they mean different things.

Reply:
Interesting, I have never seen that before... openEO only uses Location right now.

Reply:
Is there an official RFC for Content-ID via HTTP Headers? I found one for Content-Location, but failed to do so for Content-ID (just found one that was related to emails).